src/azul/__init__.py

```diff
@@ -1165,6 +1167,20 @@ def is_lower_sandbox(self) -> bool:
         def is_unit_test(self):
             return self.name == self.test_name
 
+        @cached_property
+        def is_anvil(self):
+            """
+            ``True`` if this deployment is primarily for AnVIL-based catalogs.
```

Comment:
While I believe it is technically possible for a deployment to have both HCA and AnVIL catalogs, such a deployment has never existed and likely never will exist, so "primarily" is an unhelpful word to use here.

I might just remove the docstrings altogether, considering how simple the body is.